LoggingKata/TacoParser.cs

```diff
@@ -6,13 +6,43 @@
     public class TacoParser
     {
         readonly ILog logger = new TacoLogger();
-        
+
         public ITrackable Parse(string line)
         {
             logger.LogInfo("Begin parsing");
 
-            // Do not fail if one record parsing fails, return null
-            return null; // TODO Implement
+            var cells = line.Split(',');
+
+            if (cells.Length < 3)
+            {
+                return null;
+            }
+
+            // grab the latitude from your array at index 0
+            string latitude = cells[0];
+            // grab the longitude from your array at index 1
+            string longitude = cells[1];
+            // grab the name from your array at index 2
+            string name = cells[2];
+            // parse your string as a `double`
+            double lat = double.Parse(latitude);
+            double lon = double.Parse(longitude);
```

Comment:
These are liable to throw an exception, but the current test cases don't cover it.  Consider making these parses more resilient using a `TryParse`